core/commands/block.go

```diff
@@ -26,7 +26,7 @@ func (bs BlockStat) String() string {
 
 var BlockCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Manipulate raw IPFS blocks.",
+		Tagline: "Interact with raw IPFS blocks.",
```

Comment:
Consistency -- using the same verb for all the toplevel commands that contain other subcommands.